pkg/injector/pod_patch.go

```diff
@@ -360,6 +361,10 @@ func HTTPStreamRequestBodyEnabled(annotations map[string]string) bool {
 	return getBoolAnnotationOrDefault(annotations, daprHTTPStreamRequestBody, defaultDaprHTTPStreamRequestBody)
 }
 
+func placementEnabled(annotations map[string]string) bool {
+	return getBoolAnnotationOrDefault(annotations, daprEnablePlacementKey, false)
```

Comment:
I think the default should be opt-in here and set as "true". 
/cc @yaron2 @artursouza

Reply:
Done.